calculator.c

```diff
@@ -0,0 +1,114 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+#define SIZE 100
+
+int evaluation(char operator, int op1, int op2){

```

Comment:
Practice naming variables and function names such that they are self-descriptive and meaningful. This applies throughout the code, mentioning here only once.